sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -871,7 +869,6 @@ class JsonSuite extends QueryTest with SharedSQLContext with TestJsonData {
               |WHERE _unparsed IS NOT NULL
             """.stripMargin),
           Row("{") ::
-            Row("") ::
```

Comment:
> are you saying that this is necessary because empty lines will be omitted now and this is checking for their absence somehow?
Empty lines won't be skipped now. The change in JsonSuite.scala is to verify that these empty lines will be skipped rather than to be parsed  as one Row with all null values or empty string.